Validation/pyproject.toml

```diff
@@ -4,13 +4,10 @@ build-backend = "setuptools.build_meta"
 
 [project]
 name = "Validation"
-version = "0.0.1"
+version = "0.0.2"
 description = "LDMX Data Validation"
 readme = "README.md"
-requires-python = ">=3.7"
+requires-python = ">=3.9"
```

Comment:
this didnt fix it, at least I see 
```
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
numba 0.61.2 requires numpy<2.3,>=1.24, but you have numpy 2.3.2 which is incompatible.
```

Reply:
In the dev container. I agree that having a max version number is not great for the further future. But I prose the following: let's have this compatible with the dev container upto the point where we have it under ldmx-sw, and then later if we move it out of it, this issue will be solved with the min requirement on the python version

Reply:
Makes sense to me :+1:

Reply:
OK, good for me if you do it here, thanks!